backend/app/services/adapters/task_kinds/helpers.py

```
@@ -0,0 +1,690 @@
+        )
+
+
+def _create_pipeline_subtask(
+    db: Session,
+    task: Kind,
+    team: Kind,
+    team_crd: Team,
+    task_crd: Task,
+    user_id: int,
+    existing_subtasks: List[Subtask],
+    bot_ids: List[int],
```

Ruff (0.14.13)

Unused function argument: `bot_ids`

(ARG001)